e2e/po/app.skip.ts

```diff
@@ -383,8 +377,8 @@ export class SkipLogin {
   httpSouthServiceStatus() {
```

Comment:
It seems core service is verified inside this method now so method name should be changed as well to coreServiceStatus